maxim/tests/test_groq.py

```
@@ -0,0 +1,454 @@
+                temperature=0.7
+            )
+            
+            # Verify response structure
+            self.assertIsNotNone(response)
+            self.assertTrue(hasattr(response, 'choices'))
+            self.assertTrue(len(response.choices) > 0)
+            self.assertIsNotNone(response.choices[0].message.content)
+            
+            print("Async response:", response.choices[0].message.content)
+
+        except Exception as e:
```

Ruff (0.11.9)

Do not catch blind exception: `Exception`

(BLE001)